### ISSUE TITLE
Downloading to Root Folder should check if the folder is a mount point

### ISSUE THREAD

**Issue by brunson** [Sonarr/Sonarr#4978]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I get the following warning: "Download client NZBGet places downloads in the root folder /tv/. You should not download to a root folder."

My media folder is a mounted filesystem in the root directory, not a directory on my root device. 

### Expected Behavior

A quick check of /proc/mounts to determine that the downloads is not on the root filesystem could suppress the incorrect error message.

### Steps To Reproduce

1. In a docker container
2. with the config "-v /path/to/media:/mountpoint"
3. Run sonarr
4. View the error "Download client NZBGet places downloads in the root folder /mountpoint/. You should not download to a root folder." in the sonarr UI

### Environment

```markdown
- OS: 20.04
- Sonarr: 3.0.7.1477
- Docker Install: Yes
- Using Reverse Proxy: No
- Browser: Any
```

### What branch are you running?

Main

### Trace Logs?

Not important

### Anything else?

Nothing

**Comment by Taloth** [comment]
'Root folder' is the library folder where the series folders exist. Sonarr is telling you t hat you shouldn't download directly into your library, but instead download into a intermediate download folder so that Sonarr can import it appropriately.

**Comment by bakerboy448** [comment]
The book icon / wiki link for the healthcheck explains this in detail

https://wiki.servarr.com/sonarr/system#downloading-into-root-folder
